Tests/MusicXMLTests/LilyPondTests/LilyPondTestSuite.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  LilyPondTestSuite.swift
+//  MusicXMLTests
+//
+//  Created by James Bean on 9/28/19.
+//
+
+import Foundation
+
+enum LilyPondTestSuite {
+    enum Traversal: String {
+        case partwise = "Partwise"
+        case timewise = "Timewise"
+    }
+    static func url(testName: String, traversal: Traversal) -> URL {
```

Comment:
Can `traversal` not be inferred at the root node?

Reply:
Oh, interesting! Totally.

Reply:
Whoa, could be a choice root element!